.github/jobs/configure-checks/all.bats

```diff
@@ -0,0 +1,371 @@
+   assert_line "    - lib..............: /opt/domjudge/domserver/lib"
+   assert_line "    - libvendor........: /opt/domjudge/domserver/lib/vendor"
+   assert_line "    - log..............: /opt/domjudge/domserver/log"
+   assert_line "    - run..............: /opt/domjudge/domserver/run"
+   assert_line "    - sql..............: /opt/domjudge/domserver/sql"
+   assert_line "    - tmp..............: /opt/domjudge/domserver/tmp"
+   assert_line "    - webapp...........: /opt/domjudge/domserver/webapp"
+   assert_line "    - example_problems.: /opt/domjudge/domserver/example_problems"
+   assert_line " * judgehost...........: /opt/domjudge/judgehost"
+   assert_line "    - bin..............: /opt/domjudge/judgehost/bin"
+   assert_line "    - etc..............: /opt/domjudge/judgehost/etc"
+   assert_line "    - lib..............: /opt/domjudge/judgehost/lib"
+   assert_line "    - libjudge.........: /opt/domjudge/judgehost/lib/judge"
+   assert_line "    - log..............: /opt/domjudge/judgehost/log"
+   assert_line "    - run..............: /opt/domjudge/judgehost/run"
+   assert_line "    - tmp..............: /opt/domjudge/judgehost/tmp"
+   assert_line "    - judge............: /opt/domjudge/judgehost/judgings"
+   assert_line "    - chroot...........: /chroot/domjudge"
+   assert_line "    - cgroup...........: /sys/fs/cgroup"
+}
+
+@test "Prefix configured" {
+   setup
+   run q_configure --prefix=/tmp
+   refute_line " * prefix..............: /opt/domjudge"
+   refute_line " * documentation.......: /opt/domjudge/doc"
+   refute_line " * domserver...........: /opt/domjudge/domserver"
+   refute_line "    - bin..............: /opt/domjudge/domserver/bin"
+   refute_line "    - etc..............: /opt/domjudge/domserver/etc"
+   refute_line "    - lib..............: /opt/domjudge/domserver/lib"
+   refute_line "    - libvendor........: /opt/domjudge/domserver/lib/vendor"
+   refute_line "    - log..............: /opt/domjudge/domserver/log"
+   refute_line "    - run..............: /opt/domjudge/domserver/run"
+   refute_line "    - sql..............: /opt/domjudge/domserver/sql"
+   refute_line "    - tmp..............: /opt/domjudge/domserver/tmp"
+   refute_line "    - webapp...........: /opt/domjudge/domserver/webapp"
+   refute_line "    - example_problems.: /opt/domjudge/domserver/example_problems"
+   refute_line " * judgehost...........: /opt/domjudge/judgehost"
+   refute_line "    - bin..............: /opt/domjudge/judgehost/bin"
+   refute_line "    - etc..............: /opt/domjudge/judgehost/etc"
+   refute_line "    - lib..............: /opt/domjudge/judgehost/lib"
+   refute_line "    - libjudge.........: /opt/domjudge/judgehost/lib/judge"
+   refute_line "    - log..............: /opt/domjudge/judgehost/log"
+   refute_line "    - run..............: /opt/domjudge/judgehost/run"
+   refute_line "    - tmp..............: /opt/domjudge/judgehost/tmp"
+   refute_line "    - judge............: /opt/domjudge/judgehost/judgings"
```

Comment:
I think it would be better to have less `refute_line`s. If we test a few that should be enough and have less code bloat.